src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -1390,6 +1390,9 @@ class Trial(Model):
   # Additional arguments to apply if selected.
   app_args = ndb.TextProperty()
 
+  # Flags that contradict the app args (separated by spaces).
```

Comment:
nit: drop "(separated by spaces)" since now it's repeated

Reply:
done